{"version": "1", "message": "ERROR: Hidden Testcases server unreachable.", "problem_id": "m19_a1", "user_id": "saad181", "score": {"sample": {"0": {"execution_time": 0.11792588233947754, "passed": false}, "1": {"execution_time": 0.07531428337097168, "passed": false}, "2": {"execution_time": 0.04341387748718262, "passed": false}, "3": {"execution_time": 0.05026865005493164, "passed": false}, "4": {"execution_time": 0.061739206314086914, "passed": false}, "5": {"execution_time": 0.04335427284240723, "passed": false}, "6": {"execution_time": 0.04593086242675781, "passed": false}, "7": {"execution_time": 0.04352617263793945, "passed": false}, "8": {"execution_time": 0.048563480377197266, "passed": false}, "9": {"execution_time": 0.04418134689331055, "passed": false}, "10": {"execution_time": 0.045551300048828125, "passed": false}, "11": {"execution_time": 0.04783749580383301, "passed": false}, "12": {"execution_time": 0.045327186584472656, "passed": false}}}, "pylint_score": "-2/10"}